spec/haskell/src/SEL4/Model/StateData.lhs

```diff
@@ -448,3 +448,14 @@ is true using the rule cur_tcb'_cross.
 
 > cur_tcb'_asrt :: KernelState -> Bool
 > cur_tcb'_asrt _ = True
+
+Asserts for first phase, non-blocking, non-calling handle_invocation.
```

Comment:
I suppose these asserts could be used elsewhere, so maybe we want to rephrase this comment